app/src/androidTest/java/com/github/se/polyfit/MainActivityTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.se.polyfit
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class MainActivityTest {
+
+  @Test
+  fun testActivityShouldBeLaunched() {
```

Comment:
Why is this empty test here? MainActivity is not changed, nor the test is testing for anything?
<img width=70 src="https://media1.giphy.com/media/xT0xeuOy2Fcl9vDGiA/giphy-downsized-medium.gif"/>

Reply:
Haha, I don't know how that got there. It's definitely a mistake. Thanks for catching it.